pkg/daemon/ceph/osd/init.go

```diff
@@ -89,6 +90,13 @@ func copyBinary(sourceDir, targetDir, filename string) error {
 	targetPath := path.Join(targetDir, filename)
 	logger.Infof("copying %s to %s", sourcePath, targetPath)
 
+	// Check if the source path exists, and look in PATH if it doesn't
+	if _, err := os.Stat(sourcePath); err != nil {
+		if sourcePath, err = exec.LookPath(filename); err != nil {
+			return nil
```

Comment:
If the file isn't found in the path, this should return the error, not nil.

Reply:
makes sense, I'll do that.